python/doc/examples/data_analysis/distribution_fitting/plot_estimate_gev_fremantle.py

```diff
@@ -1,167 +1,470 @@
-result_rl10_prof = factory.buildReturnLevelProfileLikelihoodEstimator(sample, 10.0)
-zm = result_rl10_prof.getParameter()
-print(f"10 years return level (profile)={zm}")
+# **Return level estimate via the profile log-likelihood function of a stationary GEV**
+#
+# We can estimate the :math:`m`-block return level :math:`z_m` directly from the data using the profile
+# likelihood with respect to :math:`z_m`.
+result_zm_10_PLL = factory.buildReturnLevelProfileLikelihoodEstimator(sample, 10.0)
+zm_10_PLL = result_zm_10_PLL.getParameter()
+print(f"10-year return level (profile) = {zm_10_PLL}")
 
 # %%
-# We can see the confidence interval of :math:`z_m` is a bit smaller with this method
-result_rl10_prof.setConfidenceLevel(0.95)
-return_level_ci10 = result_rl10_prof.getParameterConfidenceInterval()
+# We can get the confidence interval of :math:`z_m`:  once more, it appears to be a bit smaller
+# than the interval obtained from the log-likelihood function.
+# As for the confidence interval of :math:`\xi`, depending on the order requested, the interval might
+# not be calculated.
+result_zm_10_PLL.setConfidenceLevel(0.95)
+try:
+    return_level_ci10 = result_zm_10_PLL.getParameterConfidenceInterval()
+except Exception as ex:
+    print(type(ex))
+    pass
+print("Maximum profile log-likelihood function : ")
 print(f"CI={return_level_ci10}")
 
 # %%
-# We can also plot the profile likelihood and the confidence interval
-view = otv.View(result_rl10_prof.drawProfileLikelihoodFunction())
+# We can also plot the profile log-likelihood function and get the confidence interval, the optimal value
+# of :math:`z_m` and its confidence interval.
+view = otv.View(result_zm_10_PLL.drawProfileLikelihoodFunction())
 
 # %%
-# Now formulate a non-stationary hypothesis for our parameters
-# First we need the grid of time values (in years)
-mesh = ot.Mesh(data[:, 0])
+# **Non stationary GEV modeling via the log-likelihood function**
+#
+# If we look at the data carefully, we see that the pattern of variation has not remained constant over
+# the observation period. There is an increase in the data through time.
+# We want to model this dependence because a slight increase in extreme sea-levels might have
+# a significant impact on the safety of coastal flood defenses.
+#
+# We have define the functional basis for each parameter of the GEV model. Even if we have
+# the possibility to affect a time-varying model to each of the 3 parameters :math:`(\mu, \sigma, \xi)`,
+# it is strongly recommended not to vary the parameter :math:`\xi` and to let it constant.
+#
+# For numerical reasons, it is strongly recommended to normalize all the data as follows:
+#
+# .. math::
+#
+#     \tau(t) = \dfrac{t-c}{d}
+#
+# where:
+#
+# - the *CenterReduce* method where :math:`c = \dfrac{1}{n} \sum_{i=1}^n t_i` is the mean time stamps
+#   and :math:`d = \sqrt{\dfrac{1}{n} \sum_{i=1}^n (t_i-c)^2}` is the standard deviation of the time stamps;
```

Comment:
This is duplicated from `GeneralizedExtremeValueFactory.buildTimeVarying()`. So I suggest to remove this and point to the API help page.

Reply:
You are right... and the text is also in the API help page. But it is really important to normalize the data and readers maight not be aware of the point. I wanted to emphasize that point. i propose to maintain the text.